app/assets/stylesheets/active_skin.sass

```diff
@@ -31,54 +31,65 @@ body.active_admin
     padding-left: 130px
     color: $skinTextColor
     text-indent: -999999px
+  .header-item
+    top: 0
   a, a:link
     color: $skinTextColor
   ul.tabs
     padding: 0
     font-size: 0
-  ul.tabs > li:hover > a, ul.tabs > li.current > a
-    color: $skinTextActiveColor
-    background: $skinActiveColor
-    @include rounded(0)
-    margin: -10px 0
-  ul.tabs > li 
-    font-size: 12px
-    margin: 0
-    a
-      padding: 13px 20px
+    & > li.menu_item
+      font-size: 12px
       margin: 0
-  ul.tabs > li.has_nested a
-    background-position: 93% 50%
-  ul.tabs > li.has_nested:hover
-    a
-      @include rounded(0)
-      background-color: $skinActiveColor
-      border: none
-      background-position: 93% 50%
-    ul
-      background: $skinActiveColor
-      @include no-shadow()
-      @include rounded(0)
-      border: none
-      li a
-        color: #fff
+      a
+        padding: 13px 20px
+        margin: 0
+      & > a
+        @include rounded(0)
+      &:hover > a, &.current > a
+        @include rounded(0)
+        border: none
+        color: $skinTextActiveColor
+        background-color: $skinActiveColor
+      ul.menu
+        @include rounded(0)
+        background: $skinActiveColor
+        a
+          color: #fff
+          &:hover
+            color: $skinTextColor
+    &#utility_nav
```

Comment:
Customization of utility naviation.

![Screenshot from 2020-04-10 14-39-13](https://user-images.githubusercontent.com/8206622/78973846-ed6ec280-7b3a-11ea-824f-023bbcfc94b5.png)